### PR TITLE
Rake task to publish empty coronavirus content item

### PR DESCRIPTION
**Pull request by hannako**
Rake task to publish coronavirus_landing_page to the Publishing API. So that we can spike a new publishing workflow. See [trello](https://trello.com/c/0uZVAYYt/35-look-at-options-for-publishers-to-update-page)

### CODE REVIEW

File: lib/tasks/publishing_api.rake [hunk 1]
```diff
@@ -2,6 +2,32 @@ require_relative "../publish_organisations_api_route"
...
 namespace :publishing_api do
+  desc "Publish coronavirus_landing_page to publishing api"
+  task publish_coronavirus_landing_page: :environment do
+    content_id = "774cee22-d896-44c1-a611-e3109cce8eae"
+    params = {
+      base_path: "/coronavirus",
+      publishing_app: "collections-publisher",
+      rendering_app: "collections",
+      public_updated_at: Time.zone.now.iso8601,
+      update_type: "major",
```
Comment (sihugh) [inline lib/tasks/publishing_api.rake]: Shall we stick with minor updates for now?

**Comment by hannako** [comment]
@sihugh would it be safer in the first instance to change the url to something different eg `/coronavirus-landing-page` and create a redirect in integration on short url manager to `/coronavirus`?

File: lib/tasks/publishing_api.rake [hunk 2]
```diff
@@ -2,6 +2,32 @@ require_relative "../publish_organisations_api_route"
...
+  desc "Publish coronavirus_landing_page to publishing api"
+  task publish_coronavirus_landing_page: :environment do
+    content_id = "774cee22-d896-44c1-a611-e3109cce8eae"
+    params = {
+      base_path: "/coronavirus",
+      publishing_app: "collections-publisher",
+      rendering_app: "collections",
+      public_updated_at: Time.zone.now.iso8601,
+      update_type: "minor",
+      schema_name: "edition",
```
Comment (sihugh) [inline lib/tasks/publishing_api.rake]: I think should be `document_type: "coronavirus_landing_page`?

Comment (sihugh) [inline lib/tasks/publishing_api.rake]: We don't need this line I think

Comment (hannako) [reply lib/tasks/publishing_api.rake]: thanks removed

Comment (hannako) [reply lib/tasks/publishing_api.rake]: @sihugh Getting `["Schema could not be validated as the schema_name was not provided"]` when running the task on integration https://deploy.integration.publishing.service.gov.uk/job/run-rake-task/132401/console I'll stick it back in!

**Pull request merged**